File: easy-solution/minesp3164/9.py
class Solution(object):
    def isPalindrome(self, x):
        if (x < 0):
            return False
        elif (x == 0):
            return True
        else:
            x = str(x)
            straight = [char for char in x]
            straight_length = len(straight)
            reverce = list(range(straight_length))
            for i in range(straight_length):
                straight[i] = int(straight[i])
            for i in range(straight_length):
                reverce[i] = straight[straight_length - 1 - i]
                if straight[i] != reverce[i]: return False
            return True

File: easy-solution/minesp3164/java/9.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x < 0){
            return false;
        }
        if(x == 0){
            return true;
        }
        else{
            int start = 0;
            int end = x.length()-1;
            strx = str(x);
            while (start < end){
                if (strx.charAt(start++) != strx.charAt(end--)) return false;
            }
            return true;
        }
    }
}